### PR TITLE
Add Peter's FDTD tests.

### LINKED ISSUE #140: New FDTD tests
Here are tests for the FDTD functionality:

arc_13: 3D
arc_12: 2D

Both have been developed so as to be fast tests rather than physically meaningful. 

In both cases, the matlab script run_fdtd_bscan creates the test output.

Both tests can be downloaded from UCL dropbox with the following information:

%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%
1.  arc_12.zip:
    You will need the claim passcode '77df7451' 
to retrieve the file(s).  

pick-up URL: https://wwwapps-live.ucl.ac.uk/cgi-bin/dropbox/dropbox-https.cgi?state=pickup_info&id=36265c34
claim id: 36265c34

%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%
1.  arc_13.zip:
    You will need the claim passcode '44c99120' 
to retrieve the file(s).  

    
pick-up URL: https://wwwapps-live.ucl.ac.uk/cgi-bin/dropbox/dropbox-https.cgi?state=pickup_info&id=e2740b4
claim id: e2740b4

### PR DESCRIPTION
**Pull request by samcunliffe**
Two more boilerplate-thonic test scripts. All of this should be improved with #101 but prefer to get these tests in early, and deboilerplate later.

Particularly would like these tests running for #139.

***

Fix #140

### CODE REVIEW

File: tdms/tests/system/test_arc12.py
Comment (samcunliffe) [inline tdms/tests/system/test_arc12.py:10]:
```suggestion
    download_data("https://zenodo.org/record/7233444/files/arc_12.zip", to=ZIP_PATH)
```

File: tdms/tests/system/test_arc13.py
Comment (samcunliffe) [inline tdms/tests/system/test_arc13.py:10]:
```suggestion
    download_data("https://zenodo.org/record/7233118/files/arc_13.zip", to=ZIP_PATH)
```

**Review by willGraham01 — APPROVED** [review]
Looks good, pending the latest MacOS oddity.

Just a thought: we're fast accruing `test_arc0x.py` files that are functionally the same (barring the input commands to tdms). Is it worth us trying to find a way to reduce the number of files? EG Making a class with the information unique to each test, then parametrising the tests accordingly? (Excuse the terrible pseudocode below)

```python

class test_info:

    reference_output = HD5File(...)
    input_file_str = "input/file/path"
    output_file_name = "name/of/output/to/compare/to/reference"

    def __init__(self, ...):
        # suitable constructor method

# some more python code...

test_cases = # a dict/list/array of test_info instances

@pytest.mark.parametrize("test", test_cases.values())
def test_fs(test):
    run_tdms(test.input_file_str, test.output_file_name)
    assert HDF5File(test.output_file_name).matches(test.reference)
```

**Comment by samcunliffe** [comment]
It's not only a good idea, it's a draft PR: #101 

@dstansby made a very nice start. His feature is actually blocked by me doing some uploads and, separately, by #123.
Updating and extending 101 can happen after #142 → #139 is merged. ⛓️

**Pull request merged**
